src/bin/cargo/commands/uninstall.rs

```diff
@@ -5,7 +5,14 @@ use cargo::ops;
 pub fn cli() -> Command {
     subcommand("uninstall")
         .about("Remove a Rust binary")
-        .arg(Arg::new("spec").value_name("SPEC").num_args(0..))
+        .arg(
+            Arg::new("spec")
+                .value_name("SPEC")
+                .num_args(0..)
+                .add::<clap_complete::ArgValueCandidates>(clap_complete::ArgValueCandidates::new(
+                    || get_installed_crates(),
```

Comment:
Can't we change this to
```suggestion
                    get_installed_crates
```